.nycrc.json

```diff
@@ -4,6 +4,7 @@
   ],
   "exclude": [
     "src/rpc.js",
+    "src/key_store/local_key_store.js",
```

Comment:
you can use noEmitHelpers/importHelpers options to pass the code coverage check.
see http://typescript.ninja/typescript-in-definitelyland/tsc-options.html

Reply:
Thanks, I've fixed the issue as you have shown.